### PR TITLE
Added TFT_eSPI dependencies on .github workflow and properties

### PR DESCRIPTION
**Pull request by Rocketct**
Added TFT_eSPI dependencies on .github workflow and properties

### CODE REVIEW

**Review by per1234 — APPROVED** [review]
The changes are fine, but the maintainer to should be sure to only merge this pull request if https://github.com/arduino-libraries/Arduino_MKRIoTCarrier/pull/36 is also accepted, since the changes made here are only necessary in that case.

It would make more sense to do these changes as part of https://github.com/arduino-libraries/Arduino_MKRIoTCarrier/pull/36, since that would allow the CI system to facilitate the review process for that PR.

**Comment by Rocketct** [comment]
ciao @per1234 yes I've spitted because i would that [#36](https://github.com/arduino-libraries/Arduino_MKRIoTCarrier/pull/36), sett all passed before to merge, next time will be all in one unique PR :smile:

**Pull request merged**
